saleor/plugins/webhook/plugin.py

```diff
@@ -37,6 +47,13 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.active = True
 
+    def check_plugin_id(self, plugin_id: str) -> bool:
+        is_webhook_plugin = super().check_plugin_id(plugin_id)
+        if not is_webhook_plugin:
```

Comment:
Before we enter to this condition, we should check the structure of plugin_id. LIke - make sure that the plugin_id is in a structure like "app:<id>:<something>. Now we will call `from_payment_app_id` for all plugins different than WebhookPlugin.

Reply:
We will not - all other plugins have a different implementation of this method where we only compare passed `plugin_ID` with `self.PLUGIN_ID`. Only WebhookPlugin has a custom method to check the ID and only this plugin can run `from_payment_app_id`.

Reply:
👍